### PR TITLE
Clone s25

### PR DESCRIPTION
**Pull request by Riaboshtan**

### CODE REVIEW

File: apps/kindergarten/urls.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+from django.conf.urls import url
+
+from apps.kindergarten.views import KinderView
+
+urlpatterns = [
+    url(r'^(?P<pk>[0-9]+)$', KinderView.as_view()),
```
Comment (aodarc) [inline apps/kindergarten/urls.py]: add name for url 
like this https://docs.djangoproject.com/en/1.11/topics/http/urls/#examples

File: apps/kindergarten/views.py [hunk 2]
```diff
@@ -1 +1,19 @@
...
+from apps.kindergarten.models import (
+    Kindergarten,
+    Group
+)
+from apps.kindergarten.templates import *
+
+from django.views.generic.detail import DetailView
+
+class KinderView(DetailView):
+    template_name = 'show_kindergarten.html'
```
Comment (aodarc) [inline apps/kindergarten/views.py]: move this template to kindergarten folder (in 'templates' folder)

**Review by aodarc — CHANGES_REQUESTED** [review]
check comments

File: apps/kindergarten/views.py (hunk 2 above)
Comment (Riaboshtan) [reply apps/kindergarten/views.py]: sorry, i don't understand what and where to move

Comment (Riaboshtan) [reply apps/kindergarten/views.py]: show_kindergatren is in apps/kindergarten/templates, no?

**Review by aodarc — APPROVED** [review]

**Review by maziar0andriy — APPROVED** [review]

**Pull request merged**